src/equations/acoustic_perturbation_2d.jl

```diff
@@ -340,6 +340,71 @@ end
 end
 
 
+# Calculate 1D flux for a single point in the normal direction
+# TODO: Better comment, here this is really a normal vector
+@inline function flux(u, normal_vector::AbstractVector, equations::AcousticPerturbationEquations2D)
+  v1_prime, v2_prime, p_prime = cons2state(u, equations)
+  v1_mean, v2_mean, c_mean, rho_mean = cons2mean(u, equations)
+
+  f1 = normal_vector[1] * (v1_mean * v1_prime + v2_mean * v2_prime + p_prime / rho_mean)
+  f2 = normal_vector[2] * (v1_mean * v1_prime + v2_mean * v2_prime + p_prime / rho_mean)
+  f3 = c_mean^2 * rho_mean * (  normal_vector[1] * (v1_prime + v1_mean * p_prime)
+                              + normal_vector[2] * (v2_prime + v2_mean * p_prime) )
```

Comment:
This doesn't look correct to me. When we set `normal_vector = (1.0, 0.0)`, we get
```julia
f3 = c_mean^2 * rho_mean * (v1_prime + v1_mean * p_prime)
```
That's not the Cartesian variant
```julia
f3 = c_mean^2 * rho_mean * v1_prime + v1_mean * p_prime
```
Hence, I think this should be something like
```suggestion
  f3 = ( normal_vector[1] * (c_mean^2 * rho_mean * v1_prime + v1_mean * p_prime)
       + normal_vector[2] * (c_mean^2 * rho_mean * v2_prime + v2_mean * p_prime) )
```